robot_smach_states/src/robot_smach_states/manipulation/manipulation.py

```diff
@@ -262,80 +277,88 @@ def __init__(self, side, x, y, z, roll=0, pitch=0, yaw=0, time_out=20, pre_grasp
         self.x = x
         self.y = y
         self.z = z
-        self.roll  = roll
+        self.roll = roll
         self.pitch = pitch
-        self.yaw   = yaw
-        self.time_out   = time_out
+        self.yaw = yaw
+        self.time_out = time_out
         self.pre_grasp = pre_grasp
-        self.frame_id  = frame_id
+        self.frame_id = frame_id
         self.delta = delta
```

Comment:
I believe the entire ArmToUserPose class is deprecated. It is not used in any of the challenges and it still uses the deprecated amigo API. Should this function be updated or removed entirely?

Reply:
Removed